lib/reverse_string.rb

```diff
@@ -1,5 +1,21 @@
 def reverse_characters(str)
+#Ok this is a little better but im still not sure how to use the .join to get it back to a string
+#The array can be called back backwards but its not "saved" anywhere in that order so it cant be converted to a string
+string="Today is a good day"
+array=string.split
+x=0
+for loop in array
+puts array[x-=1]
 end
```

Comment:
This `end` is ending the `for` loop, but where did the `end` for the method itself go? I think you accidentally lost it in the coding process :)